tools/pipelines/publish-api-model-artifact.yml

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# publish-api-model-artifact pipeline
+# This pipeline downloads the api-extractor artifacts from build pipelines and uploads them to the azure blob storage.
+# A check version stage is used to determine if the branch should be deployed.
+# If the release branch is the latest minor version of its major version series, it should be deployed.
```

Comment:
Nit: I don't think we need to explicitly note the implementation details here (the check version stage). Instead, we should be able to just say something like:
```suggestion
# Artifacts are only published from the latest minor release branch of each major version series.
```

Maybe could note the expected release branch naming pattern.

Reply:
Probably also worth noting that we trigger the website deployment pipeline after publishing an updated artifact.